doc/cephadm/octopus_gsg.rst

```diff
@@ -0,0 +1,262 @@
+.. _octopus_gsg:
+
+Installing Ceph Octopus
+=======================
+
+This procedure is a step-by-step walkthrough of the installation of a three-node Ceph clutser. If you have never installed Ceph before and you don't know where to begin, begin here.
+
+Removing an old cluster
+-----------------------
+
+If you mess up the installation at any point during the procedure, you can run this two-step procedure to return your system to the beginning state. If you mess up or you lose your place or you just want to start over from the beginning, now you know how.
+
+1. Run ``cephadm ls`` and make a note of the filesystem ID (fsid)::
```

Comment:
```suggestion
#. Run ``cephadm ls`` and make a note of the filesystem ID (fsid)::

Reply:
Numbering in RST is done with a hash so that it is auto-enumerated: https://docutils.sourceforge.io/docs/user/rst/quickref.html#enumerated-lists

Reply:
I have not had good luck with auto-numbering in RST, but I'm willing to give it another go.

Reply:
Really? That's odd :thinking: the compiler?